python/ray/tune/tests/test_trial_scheduler_pbt.py

```diff
@@ -151,6 +201,25 @@ def testPermutationContinuation(self):
             stop={"training_iteration": 3})
 
     def testPermutationContinuationFunc(self):
+        def MockTrainingFunc(config, checkpoint_dir=None):
+            iter = 0
+            a = config["a"]
+            b = config["b"]
+
+            if checkpoint_dir:
+                checkpoint_path = os.path.join(checkpoint_dir, "model.mock")
+                with open(checkpoint_path, "rb") as fp:
+                    a, b, iter = pickle.load(fp)
+
+            while True:
+                iter += 1
+                with tune.checkpoint_dir(step=iter) as checkpoint_dir:
+                    checkpoint_path = os.path.join(checkpoint_dir,
+                                                   "model.mock")
+                    with open(checkpoint_path, "wb") as fp:
+                        pickle.dump((a, b, iter), fp)
+                tune.report(mean_accuracy=(a - iter) * b)
```

Comment:
btw, why do we have to change these tests?

Reply:
I just moved the trainables to inside the test functions, I didn't actually change them. The diff shows up weird though.